target_chains/cosmwasm/examples/cw-contract/Cargo.toml

```diff
@@ -34,5 +34,5 @@ cosmwasm-storage = { version = "1.0.0" }
 cw-storage-plus = "0.13.4"
 schemars = "0.8"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
-pyth-cosmwasm = { git="https://github.com/pyth-network/pyth-crosschain", tag="pyth-cosmwasm-v0.1.0", features=["library"] }
+pyth-sdk-cw = { path="../../pyth-sdk-cw"}
```

Comment:
It would be good if you also add a version here so we get error if they are not a match and also add a comment that people need to use the deployed version when they copy/paste the example.

```
pyth-sdk-cw = { path="..", version="x.y.z" }
```

Reply:
Yes, in the next PR. I am going to release the module and will update this part to consume that.